Zend/zend_builtin_functions.c

```diff
@@ -763,7 +763,7 @@ ZEND_FUNCTION(each)
 		if (Z_REFCOUNTED_P(entry)) Z_ADDREF_P(entry);
 	}
 	zend_hash_index_add_new(Z_ARRVAL_P(return_value), 1, entry);
-	zend_hash_add_new(Z_ARRVAL_P(return_value), CG(known_strings)[ZEND_STR_VALUE], entry);
+	zend_hash_add_new(Z_ARRVAL_P(return_value), ZSTR_KNOWN(ZEND_STR_VALUE), entry);
```

Comment:
CG(known_strings) is now removed, instead it's a real global var zend_known_strings. Same is with zend_empty_string and zend_one_char_string. So we would reference a global var directly, which is worse than having an acessor macro. But in general, it is a a change that belongs to the patch, so either way it were not possible to keep the exact current code.